src/App.jsx

```diff
@@ -14,45 +16,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_ALPHABETICALLY = 'sort alphabetically';
+const SORT_BY_LENGTH = 'sort by length';
+
+const sortByCondition = (condition, isReversed) => {
+  const copyGoods = [...goodsFromServer];
+
+  if (condition) {
+    copyGoods.sort((good1, good2) => {
+      switch (condition) {
+        case SORT_ALPHABETICALLY:
+          return good1.localeCompare(good2);
+
+        case SORT_BY_LENGTH:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  return isReversed ? copyGoods.reverse() : copyGoods;
```

Comment:
![image](https://github.com/mate-academy/react_list-of-goods-js/assets/128422319/2c6ae9b0-5de1-4ec5-87a8-9c61835ca38c)